integrations/airflow/setup.py

```diff
@@ -20,9 +20,10 @@
     readme = readme_file.read()
 
 requirements = [
-    "attrs>=19.3",
+    "attrs==19.3.0",
```

Comment:
I'm wondering if we want to fix this version?

Reply:
Sorry- that was done to verify the cause of a particular problem. I think we can get away without this